docs/vale/styles/config/vocabularies/Dagster/accept.txt

```diff
@@ -38,5 +38,10 @@ Twilio
 
 We have
 we have
+
+[Ww]eb server
+Dockerfile
+AWS
+ECS
+
 lookback
-Pydantic
```

Comment:
@cmpadden was removing Pydantic on purpose?

Reply:
Whoops, it was not. I can re-introduce.